Sources/App/Models/Manifest.swift

```diff
@@ -37,10 +37,17 @@ struct Manifest: Decodable, Equatable {
         var name: String
         var type: `Type`
     }
+    struct ToolsVersion: Decodable, Equatable {
+        enum CodingKeys: String, CodingKey {
+            case version = "_version"
```

Comment:
I'll need to check but I think we've got a fixture from an actual manifest that we should extend to test for this property name. I believe the existing tests that you modified injected the tools version rather than use actual output, so it's not testing the property name.